src/configuration/app-containers.md

```diff
@@ -51,59 +52,59 @@ web:
 disk: 2048
 
-# The 'mounts' describe writable, persistent filesystem mounts in the application. The keys are
-# directory paths, relative to the application root. The values are strings such as
-# 'shared:files/NAME', where NAME is just a unique name for the mount.
+# The 'mounts' describe writable, persistent filesystem mounts in the application.
+# The keys are directory paths relative to the application root. The values are
+# strings such as 'shared:files/NAME' where NAME is just a unique name for the mount.
 mounts:
     '/web/files': 'shared:files/web-files'
 
-# The hooks that will be performed when the package is deployed.
+# The hooks that will be triggered when the package is deployed.
 hooks:
-    # Build hooks can modify the application files on disk, but not access any services like databases.
+    # Build hooks can modify the application files on disk but not access any services like databases.
     build: |
       rm web/app_dev.php
-    # Deploy hooks can access services, but the file system is now read-only.
+    # Deploy hooks can access services but the file system is now read-only.
     deploy: |
       app/console --env=prod cache:clear
 ```
 
 > **Note**
 > This configuration file is specific to one application. If you have multiple
-> applications inside your Git repository (i.e. a RESTful
-> web service and a front-end, or a main web site and a blog), you need
-> one `.platform.app.yaml` at the root of each application. See the [Multi-app](/configuration/app/multi-app.md) documentation.
+> applications inside your Git repository (i.e. a RESTful web service and a
+> front-end or a main web site and a blog), you need one `.platform.app.yaml`
+> at the root of each application. See the [Multi-app](/configuration/app/multi-app.md) documentation.
 
 ### Name
 
 The `name` is the unique identifier of the application. Platform.sh
 supports multiple applications within a project, so each application
 must have a **unique name** within a project. The name may only be
-composed of lower case alpha-numeric characters. (a-z0-9).  *Be advised
-that changing the `name` of your app after it's been deployed will
-destroy all storage volumes, and thus is typically a Very Bad Thing to do.*
+composed of lower case alpha-numeric characters (a-z0-9).  *Be advised
+that changing the `name` of your application after it has been deployed will
+destroy all storage volumes and thus, is typically a Very Bad Thing to do.*
 It could be useful under certain circumstances in the early stages
-of development but you almost certainly don't want to change it on 
+of development but you almost certainly don't want to change it on
 a live project.
 
 This name is used in the `.platform/routes.yaml` file to define the HTTP upstream
-(by default `php:http` - if you called your application `app` you will
+(by default `php:http`; if you called your application `app` you will
 need to use `app:http` in the upstream field).
```

Comment:
I agree the paranthesis should be removed and replaced by a new sentence instead.